libsnark/zk_proof_systems/plonk/tests/bls12_381_test_vector_transcript_hasher.hpp

```diff
@@ -0,0 +1,123 @@
+/** @file
+ *****************************************************************************
+ * @author     This file is part of libff, developed by Clearmatics Ltd
+ *             (originally developed by SCIPR Lab) and contributors
+ *             (see AUTHORS).
+ * @copyright  MIT license (see LICENSE file)
+ *****************************************************************************/
+
```

Comment:
On my machine (using clang), I need `#include <array>` in this file to make the build work.

Reply:
Added.